unity_jobs_simulator/Assets/Scripts/Robot_Controller.cs

```diff
@@ -0,0 +1,273 @@
+        }
+        if(Input.GetKeyDown(KeyCode.S))
+        {
+            TargetJointAngles = new List<float>(){-45, -30, 15, 0, 90, 90};
+        }
+        if(Input.GetKeyDown(KeyCode.Keypad1) || Input.GetKeyDown(KeyCode.Alpha1))
+        {
+            TargetJointAngles = new List<float>() {90, 0, 0, 0, 0, 0};
+        }
+        if(Input.GetKeyDown(KeyCode.Keypad2) || Input.GetKeyDown(KeyCode.Alpha2))
+        {
+            TargetJointAngles = new List<float>() {0, -90, 0, 0, 0, 0};
+        }
+        if(Input.GetKeyDown(KeyCode.Keypad3) || Input.GetKeyDown(KeyCode.Alpha3))
+        {
+            TargetJointAngles = new List<float>() {0, 0, -90, 0, 0, 0};
+        }
+        if(Input.GetKeyDown(KeyCode.Keypad4) || Input.GetKeyDown(KeyCode.Alpha4))
+        {
+            TargetJointAngles = new List<float>() {0, 0, 0, 90, 0, 0};
+        }
+        if(Input.GetKeyDown(KeyCode.Keypad5) || Input.GetKeyDown(KeyCode.Alpha5))
+        {
+            TargetJointAngles = new List<float>() {0, 0, 0, 0, -90, 0};
+        }
+        if(Input.GetKeyDown(KeyCode.Keypad6) || Input.GetKeyDown(KeyCode.Alpha6))
+        {
+            TargetJointAngles = new List<float>() {0, 0, 0, 0, 0, 90};
+        }
+    }
+
+    void FixedUpdate()
+    {
+        UpdateCurrentDrivePos(); // We need to update the current position of the Robot to ensure we know exactly where it is
+        
+        if(SamePos(CurrentJointAngles, TargetJointAngles)) // If we are at the preset target, FixedUpdate shouldn't run so we return nothing
+        {
+            if(CheckAngleError)
+            {
+                CheckAngleErrors();
+            }
+            return;
+        }
+
+        if(!(SamePos(_currentDriveTarget, _enroute))) // If the route has changed, we need to restart the motion
+        {
+            _enroute = new List<float>(TargetJointAngles);
+            _incAngles = new List<float>(_zeroList);
+        }
+
+        if(SamePos(_incAngles, _zeroList)) // If we don't have increments for the angles, then we aren't going anywhere and need to initialize these increments
+        {
+            _maxVelRad = MaxVelocityDeg * Mathf.Deg2Rad; // This updates the value of _maxVelRad if the input MaxVelocityDeg is changed during motion
+            GetIncrements(); // This calculates the values for increments given the target angles, the current positions, and the maximum velocity
+            SetIncAngles(); // This calculates the incremental angles given the increments and the target angles
+            _enroute = new List<float>(TargetJointAngles); // We set our enroute position as this is where our robot movement will be moving to
+            _nextAngles = AddLists(_currentDriveTarget, _incAngles); // This increments the angles by one increment
+        }
+
+        if(SamePos(_nextAngles, TargetJointAngles)) // If the next Angle is the final target position, then this is the final increment
```

Comment:
What happens if `_nextAngles` exceeds the `TargetJointAngles`? You should be checking if the max joint angle error is greater than or equal to the target, and then set final values to the target. 

Doing this might allow you to set a smaller angle sensitivity.